lib/StyleDictionary.js

```diff
@@ -541,35 +585,25 @@ export default class StyleDictionary extends Register {
         console.log(chalk.rgb(255, 140, 0).bold(err));
       }
     }
-
-    return exportableResult;
+    return { tokens: exportableResult, allTokens: flattenTokens(exportableResult, this.usesDtcg) };
```

Comment:
We flatten these only once now after computing the transformed tokens for platform, instead of doing it like 3 times before this change, saves a lot of compute for large sets.